### PR TITLE
Update Get-CsOnlineUser.md

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -103,6 +104,13 @@ Example 5 returns information for user accounts that have been assigned a design
 
 ## PARAMETERS
 
+### -AccountType
+New parameter added for Get-CsOnlineUser which indicates the user type. The possible values for AccountType are: 
+- 'User' - to query for user accounts
+- 'ResourceAccount' - to query for app endpoints or resource accounts
+- 'Guest' - to query for guest accounts
+- 'Unknown' - to query for user type that is not known
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:112]:
```suggestion
- `User` - to query for user accounts.
- `ResourceAccount` - to query for app endpoints or resource accounts.
- `Guest` - to query for guest accounts.
- `Unknown` - to query for a user type that is not known.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -103,6 +104,13 @@ Example 5 returns information for user accounts that have been assigned a design
 
 ## PARAMETERS
 
+### -AccountType
+New parameter added for Get-CsOnlineUser which indicates the user type. The possible values for AccountType are:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:108]:
```suggestion
This parameter is added to Get-CsOnlineUser to indicate the user type. The possible values for the AccountType parameter are: 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by dariomws** [comment]
Hi @isrumnon, when will this parameter be available? Next module release? I cannot see it in 4.5.0. Thanks!

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by isrumnon** [comment]
> Hi @isrumnon, when will this parameter be available? Next module release? I cannot see it in 4.5.0. Thanks!

Hi @dariomws are you referring to the onprem parameters?

**Comment by dariomws** [comment]
@isrumnon, I'm referring to AccountType.

image

I also just noticed the new in brackets. I think we should remove it.

**Comment by tiburd** [comment]
@dariomws 
Hi!
I assume this PR is still in progress...?
Please @ mention me when it's ready to be merged.
Thank you!

**Comment by dariomws** [comment]
@tiburd, I made a suggestion to correct something minor, but I think it would be best to wait yes.

@isrumnon, could you review it?

**Comment by yogkumgit** [comment]
@dariomws Thanks for the review. A small request, please approve the PR only when your suggested comments are implemented by Author. Thanks

**Comment by isrumnon** [comment]
@dariomws AccountType is available from 4.5.1 onwards. I'll state the same in the documentation

**Comment by dariomws** [comment]
Thanks, @isrumnon.

I read "This parameter is added to Get-CsOnlineUser starting from TPM 2.6.0 to indicate the user type". It should be 4.5.1.

**Comment by isrumnon** [comment]
Corrected it. Thanks!

**Review by dariomws — APPROVED** [review]

**Comment by tiburd** [comment]
@dariomws 
If this PR is approved and ready to merge now, please @ mention me.
Thank you!

**Comment by dariomws** [comment]
@tiburd, yes, it's ready to merge!

**Pull request merged**